lib/widgets/message_list.dart

```diff
@@ -1092,11 +1093,13 @@ class StreamMessageRecipientHeader extends StatelessWidget {
       child: Row(
         children: [
           Flexible(
-            child: Text(topic.displayName,
+            child: Text(topic.displayName ?? store.realmEmptyTopicDisplayName,
               // TODO: Give a way to see the whole topic (maybe a
               //   long-press interaction?)
               overflow: TextOverflow.ellipsis,
-              style: recipientHeaderTextStyle(context))),
+              style: recipientHeaderTextStyle(context).copyWith(
+                fontStyle: topic.displayName == null ? FontStyle.italic : null,
```

Comment:
Instead of copying the style object right after making it, let's have this `recipientHeaderTextStyle` helper take a parameter. Can just be `fontStyle` directly.